refarch-integrations/refarch-dms-integration/refarch-dms-integration-fabasoft/refarch-dms-integration-fabasoft-mock-service/src/main/java/de/muenchen/refarch/integration/dms/fabasoft/mock/service/FabasoftMockApplication.java

```diff
@@ -4,6 +4,7 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
+@SuppressWarnings("PMD.UseUtilityClass")
```

Comment:
Instead of adding the suppression I'd add a private constructor that disables instantiation of the class.

Reply:
Last time you sad explicitly to suppress it instead of a private constructor :D
See https://github.com/it-at-m/refarch-templates/blob/main/refarch-backend/src/main/java/de/muenchen/refarch/MicroServiceApplication.java